docs/en/index.md

```diff
@@ -0,0 +1,83 @@
+# TOTP authenticator
+
```

Comment:
Please make this consistent with the webauthn docs - either have installation first or have a preamble about the authenticator first, but they should be consistent.

Reply:
Sorry didn't push

Reply:
Have moved installation first in webauthn

There wasn't a preamble for totp. I'm not going to write one at this point, because I'm only restructuring things and deleting outdated content.